Lib/Compiler.py

```diff
@@ -88,7 +88,7 @@ def __compileApp( self, extPath, extName, appName, extClassPath = "" ):
     classPath.append( os.path.join( extPath, appName, "classes" ) )
     
     cmd = [ 'sencha', '-sdk', self.__sdkPath, 'compile', '-classpath=%s' % ",".join( classPath ),
-           '-debug=%s' % self.__debugFlag, 'page', '-name=page','-in',inFile, '-out', outFile,'and',
+           '-debug=%s' % self.__debugFlag, 'page', '-name=page','-input-file',inFile, '-out', outFile,'and',
```

Comment:
--input-file 
In addition if you change that one, change -out as well: --output-page
Please not:
-input-file is equal to -i option
-in as well.
The options are:
--input-file or -i
--output-page or -ou
It works using -in or input-file, because of backward compatibility. If you write input-file I really recommend -- option, because sencha can change the current behaviour.